### DISCUSSION TITLE
sock.c:344  UCX  ERROR recv(fd=47) failed: Connection reset by peer

### DISCUSSION THREAD

**Discussion by smallriver666** [openucx/ucx#8911] [Q&A]
I encountered this problem when submitting tasks with multiple nodes. I have a management node and five computing nodes. Nodes 2 to 5 are the same server. Node 6 is not the same server as the other four nodes. I am on 2345 four Tasks can be successfully submitted on nodes, and the following error will occur when node 6 is added (I am using openmpi4.1.0)
![image](https://user-images.githubusercontent.com/104721358/222112043-d1e03ebf-1cd9-4d31-8108-86e5a235125f.png)
This is the script I used to submit the task:
![image](https://user-images.githubusercontent.com/104721358/222112429-d9655355-375e-447e-a1da-ee02d5f0545f.png)

**Comment by yosefe** [comment]
can you pls add `-x UCX_LOG_LEVEL_TRIGGER=error -x UCX_HANDLE_ERRORS=bt,freeze` to mirin command, and post the printed backtrace?
